common/av_log.c

```diff
@@ -206,14 +206,13 @@ void check_library_versions(struct mp_log *log, int v)
     };
 
     mp_msg(log, v, "FFmpeg version: %s\n", av_version_info());
-    mp_msg(log, v, "FFmpeg library versions:\n");
+    mp_msg(log, v, "FFmpeg library versions:");
 
     for (int n = 0; n < MP_ARRAY_SIZE(libs); n++) {
         const struct lib *l = &libs[n];
-        mp_msg(log, v, "   %-15s %d.%d.%d", l->name, V(l->buildv));
+        mp_msg(log, v, "\n   %-15s %d.%d.%d", l->name, V(l->buildv));
```

Comment:
Why? This code is correct, no?

Why not remove the superfulus new line itself?
```diff
diff --git a/player/main.c b/player/main.c
index b615cf9e65..a11b9dff87 100644
--- a/player/main.c
+++ b/player/main.c
@@ -155,7 +155,6 @@ void mp_print_version(struct mp_log *log, int always)
         mp_msg(log, v, " built on %s\n", mpv_builddate);
     mp_msg(log, v, "libplacebo version: %s\n", PL_VERSION);
     check_library_versions(log, v);
-    mp_msg(log, v, "\n");
     // Only in verbose mode.
     if (!always) {
         mp_msg(log, MSGL_V, "Configuration: " CONFIGURATION "\n");
```

Reply:
You're right, changed.